src/devcards/core.clj

```diff
@@ -178,13 +179,18 @@
   (when (utils/devcards-active?)
     `(devcards.util.utils/pprint-code ~obj)))
 
-(defmacro mkdn-code [body] `(str "\n```\n" ~body "```\n"))
+(defmacro mkdn-code [body] `(str "\n```clojure\n" ~body "\n```\n"))
 
 (defmacro mkdn-pprint-code [obj]
   (when (utils/devcards-active?)
     `(mkdn-code
       (devcards.util.utils/pprint-code ~obj))))
 
+(defmacro mkdn-pprint-source [obj]
+  (when (utils/devcards-active?)
+    `(mkdn-code
+       ~(or (cljs.repl/source-fn &env obj) (str "Source not found")))))
+
```

Comment:
I followed your naming convention for this new macro. I also thought about naming it simply `source` to keep it short on the user side of things as just `(dc/source foo)` but decided to defer that decision to you. Just a thought.